deploy/olm-catalog/ibm-block-csi-operator/1.5.1/bundle-1.5.1.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1="maintenance-1.5"
+LABEL operators.operatorframework.io.bundle.channels.v1="maintenance-1.5"
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.6,v4.7"
+LABEL com.redhat.delivery.operator.bundle=true
+LABEL com.redhat.delivery.backport=true
```

Comment:
in our certified versions, I think @barakda decided to remove this label.
so we should consider to do the same for community (at least as of 1.6.0, not sure about 1.5.1 though)

Reply:
I see it was already done in the work in progress for community 1.6.0.
let's leave it as is here